consensus/spos/consensusState_test.go

```diff
@@ -4,10 +4,8 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/ElrondNetwork/elrond-go/consensus"
```

Comment:
This testFile should be changed to use bls everywhere instead of bn. The tests are not inside the bn folder so they should still remain but with bls.

Reply:
done.

Reply:
ok